From 7e3b4b2fca44d37549f9431d9d5259d02e42ee83 Mon Sep 17 00:00:00 2001
From: Rex Zhu <Rex.Zhu@amd.com>
Date: Fri, 21 Apr 2017 18:33:05 +0800
Subject: [PATCH 1288/2094] drm/amd/powerplay: disable cks by default on
 vega10.

run gpu test auto reboot when enable cks right now.

Change-Id: Iea5bf3b5cdc198efc3ca5a7336bb15a5eaab7e1d
Signed-off-by: Rex Zhu <Rex.Zhu@amd.com>
Reviewed-by: Alex Deucher <alexander.deucher@amd.com>
---
 drivers/gpu/drm/amd/powerplay/hwmgr/vega10_hwmgr.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/powerplay/hwmgr/vega10_hwmgr.c b/drivers/gpu/drm/amd/powerplay/hwmgr/vega10_hwmgr.c
index 5bb18a9..be2b602 100644
--- a/drivers/gpu/drm/amd/powerplay/hwmgr/vega10_hwmgr.c
+++ b/drivers/gpu/drm/amd/powerplay/hwmgr/vega10_hwmgr.c
@@ -124,7 +124,7 @@ static void vega10_set_default_registry_data(struct pp_hwmgr *hwmgr)
 	}
 
 	data->registry_data.clock_stretcher_support =
-			hwmgr->feature_mask & PP_CLOCK_STRETCH_MASK ? true : false;
+			hwmgr->feature_mask & PP_CLOCK_STRETCH_MASK ? false : true;
 
 	data->registry_data.disable_water_mark = 0;
 
-- 
2.7.4

